server/routes/ivory-age.route.js

```diff
@@ -0,0 +1,125 @@
+'use strict'
+
+const { Paths, RedisKeys, Views } = require('../utils/constants')
+const RedisService = require('../services/redis.service')
+const { buildErrorSummary, Validators } = require('../utils/validation')
+
+const itemType = 'musical' // temporary to deal with dynamic heading until parent page built. Values: 'musical', '10%' or 'other'
```

Comment:
Create a const lookup object for these values to avoid use of magic strings,

e.g. 

const ItemType = {
  MUSICAL: 'musical',
  TEN_PERCENT: '10%',
  OTHER: 'other'
}